CODE_OF_CONDUCT.md

```diff
@@ -35,19 +42,21 @@ The conda Organization is dedicated to providing a harassment-free community for
 
 Thank you for helping make this a welcoming, friendly community for all.
 
-**CODE OF CONDUCT REPORTING FORM:** [*https://URL/to/your/CoC/reporting/form*](https://URL/to/your/CoC/reporting/form)
+## [Report a code of conduct incident here.](https://form.jotform.com/221527028480048)
 
-## The Longer Version
+&nbsp;
 
-### conda Organization Diversity Statement
+And now the longer version...
+
+# conda Organization Diversity Statement
```

Comment:
Hm, I assumed that conda would be capitalized when used at the start of sentence and headlines etc?

Reply:
Um.... Ask @kathatherine ?

Reply:
Yeah, Jannis is right. I forgot about that capitalization rule when I was doing my initial review of this.

Reply:
Thanks @kathatherine.  I will update this too.

Reply:
That reminds me, we need a section in the docs somewhere where we spell this out in detail I think, written language is so awkward sometimes :)

Reply:
Made an issue in conda-docs: https://github.com/conda/conda-docs/issues/803

Went ahead and brought up an issue I've seen for a while when determining how to write about conda. Should try to make a for sure decision about the rules, so we can be consistent and hopefully not confuse everybody!